README.md

```diff
@@ -1,56 +1,119 @@
-[Racket](https://racket-lang.org/) is a general-purpose programming
-language and an ecosystem for language-oriented programming.
+# Racket Programming Language
 
-This repository holds the source code for the core of Racket plus some
-related packages. The rest of the Racket distribution source code is
-in other repositories, mostly under [the Racket GitHub
-organization](https://github.com/racket).
+![Build](https://img.shields.io/github/actions/workflow/status/racket/racket/ci.yml)
+![License: MIT/Apache/LGPL](https://img.shields.io/badge/license-MIT%2FApache%2FLGPL-blue)
+![Made with Scheme](https://img.shields.io/badge/Made%20with-Scheme-lightgrey?logo=racket)
 
-Quick Start
------------
+<img width="396" height="120" alt="image" src="https://github.com/user-attachments/assets/0526b430-699e-488d-9378-7040f8458c72" />
 
-Pre-built versions of Racket for a variety of operating systems and
-architectures, as well as convenient source distributions are
-available at
+**Racket** is a general-purpose, multi-paradigm programming language in the Lisp/Scheme family, and an ecosystem for language-oriented programming.
```

Comment:
I'm not sure what "multi-paradigm" means technically, and so I'd just drop that phrase. And also the comma before `and`.